DEVEL_README.md

```diff
@@ -0,0 +1,51 @@
+# Doing simple local web development
+
+### Setup your config
+Edit `combust.devel.conf` (if needed)
+
+### Edit your local /etc/hosts file, add:
+```
+127.0.0.1  wwwperl.local
```

Comment:
.local is generally used for MDNS.  I use .lh for this.

Reply:
(There is unlikely to be a conflict, but still..)

Reply:
> .local is generally used for MDNS. I use .lh for this.

All the existing docs use `.local` - so leaving as is